modules/custom/openy_style_guide/src/Controller/OpenyStyleGuideController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Drupal\openy_style_guide\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Url;
+use Drupal\Core\Link;
+
+/**
+ * Provides route responses for the Open Y Style Guide module.
+ */
+class OpenyStyleGuideController extends ControllerBase {
+
+  /**
+   * Renders content.
+   */
+  public function content() {
+    $query = \Drupal::service('entity.query')
```

Comment:
Please, use DI
More see https://www.drupal.org/docs/8/api/services-and-dependency-injection/services-and-dependency-injection-in-drupal-8

Reply:
Thanks for the review comments, I will correct it.